lib/types/operation.js

```diff
@@ -242,6 +244,10 @@ Operation.prototype.getType = function (param) {
         return ref;
       }
     } else {
+      // If inline schema, we add it our interal hash -> which gives us it's ID (int)
+      if(schema.type === 'object') {
+        return 'inline#'+ this.addInlineModel(schema);
```

Comment:
As far as I can tell (looking at lib/type/model.js ), "Inline Model" is just decoration? Which is fine...consistency is king.